### PR TITLE
Feature base template #3

### LINKED ISSUE #3: Fancy design with mobile responsive view
1. create base html5 template for polls application
2. extend base template in index.html, detail.html, result.html
3. add [bootstrap](https://getbootstrap.com/) to your base template as cdn link
4. create div container, and move all content in this container, add bootstrap classes to all html tags

### PR DESCRIPTION
**Pull request by avoevodin**
#3

### CODE REVIEW

File: mysite/polls/templates/polls/base.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="ru">
```
Comment (iliadmitriev) [inline mysite/polls/templates/polls/base.html]: Is it English?

Comment (iliadmitriev) [inline mysite/polls/templates/polls/base.html]: Isn't is English?

Comment (iliadmitriev) [reply mysite/polls/templates/polls/base.html]: @avoevodin do you see?

Comment (avoevodin) [reply mysite/polls/templates/polls/base.html]: resolved

**Review by iliadmitriev — APPROVED** [review]

**Pull request merged**